.travis.yml

```diff
@@ -1,4 +1,6 @@
 language: rust
+rust:
```

Comment:
@reem added nightly, 1.1.0 has to be defined for the stable part though .. beta is available too if wanted